components/Admin/OrderContent.js

```diff
@@ -13,7 +13,7 @@ import { PropTypes } from 'prop-types';
 import React from 'react';
 
 import { OrderPropType } from '../../globals';
-import { getSimpleDate } from '../../utils/getSimpleDate';
+import { getReadableDate } from '../../utils/getDate';
```

Comment:
```suggestion
import { getReadableDate } from '../../utils';
```
we can just import from utils if we modify the `index.js` file in utils